### DISCUSSION TITLE
How to start server programatically for integration tests

### DISCUSSION THREAD

**Discussion by brpaz** [nuxt/nuxt#19293] [Questions]
Hello.
I want to start the Nuxt server programmatically for doing integration tests of my server API endpoints. (using jest)

Is there any way to do this? I couldn´t find any updated documentation for it.

**Comment by Baroshem** [comment]
Hey,

What about using the default vitest tests for modules that are provided when building a new module https://github.com/Baroshem/nuxt-medusa/blob/main/test/basic.test.ts?

I think this could serve as integration tests as well
